### PR TITLE
Added yaml schedule file for powerVM prepare job

### PR DESCRIPTION
**Pull request by tinawang123**
Added yaml schedule file for powerVM prepare job

- Related ticket: https://progress.opensuse.org/issues/157837
- Related PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19348

### CODE REVIEW

File: schedule/yast/autoyast/autoyast_mig_spvm.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: autoyast_mig_spvm
```
Comment (lemon-suse) [inline schedule/yast/autoyast/autoyast_mig_spvm.yaml:2]: This is installation process and  prepared for migration test, how about name it as 'autoyast_install_spvm'?

File: schedule/yast/autoyast/autoyast_mig_spvm.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+description: >
+  Migration autoYaST installation and do prepare for migration standalone
```
Comment (lemon-suse) [inline schedule/yast/autoyast/autoyast_mig_spvm.yaml:4]: 'standalone' is for ha test and not for other installation, so please remove it.

File: schedule/yast/autoyast/autoyast_mig_spvm.yaml (hunk 1 above)
Comment (tinawang123) [reply schedule/yast/autoyast/autoyast_mig_spvm.yaml:2]: This schedule file includes 'patch_sle', it is not only about installation. So I think 'mig' is better.

File: schedule/yast/autoyast/autoyast_mig_spvm.yaml (hunk 2 above)
Comment (tinawang123) [reply schedule/yast/autoyast/autoyast_mig_spvm.yaml:4]: Updated.

File: schedule/yast/autoyast/autoyast_mig_spvm.yaml (hunk 1 above)
Comment (tinawang123) [reply schedule/yast/autoyast/autoyast_mig_spvm.yaml]: Updated

**Comment by lemon-suse** [comment]
CI tidy issue.

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**